app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -854,14 +856,16 @@ class PipelineSampleReport extends React.Component {
     );
   }
 
-  downloadAssemblyLink(e) {
-    const taxId = e.target.getAttribute("data-tax-id");
-    location.href = `/samples/${this.sample_id}/assembly/${taxId}`;
+  gotoTreeLink(taxid) {
+    window.open(
+      `/phylo_trees/index?taxid=${taxid}&project_id=${this.projectId}`,
+      "_blank"
+    );
```

Comment:
add "noopener", "hide_referrer" after "_blank"

https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/